docs/Privacy/Privacy.md

```diff
@@ -0,0 +1,107 @@
+## Privacy
+
+### Private Transaction
+
+A Transactions whose payload is only visible/store to the network actors
+whose public keys are located in the `privateFor` parameter, and not in
+any other node . `privateFor` can take multiple addresses in a comma
+separated list. ( [eea_sendrawtransaction](../Reference/JSON-RPC-API-Methods.md#eea_sendRawTransaction) - see an example of private
+transaction). Private transactions are kept private between related
+parties, so unrelated parties have no access to the content of the
+transaction, the sending party, or the list of participating addresses.
+
+
+### Important Concepts
+- **Precompiled Contract**:    A smart contract compiled from its source
+ language to EVM bytecode and stored by an Ethereum node for later
+ execution.
+- **Privacy**: It is “The right of individuals to control or influence
+ what information related to them may be collected and stored and by
+ whom and to whom that information may be disclosed.” For the purposes
+ of this Specification, this right of privacy also applies to
+ organizations to the extent permitted by law.
+- **Private State**:   A data that is not shared in the clear in the
+ globally replicated state tree. This data can represent bilateral or
+ multilateral arrangements between counterparties, for example in
+ private transactions.
+- **Private Transaction**: A transaction where some information about
+ the transaction, such as the payload data, or the sender and
+ recipient, is only available to the subset of network actors who are
+ parties to that transaction.
+- **Private Transaction Manager**: A subsystem of an Enterprise Ethereum
+  system for implementing privacy and permissioning known as Enclave.
+  e.g. [Orion](https://github.com/PegaSysEng/orion) .
+- **Actor**:   The public key of someone involved in the private transaction (as per the privateFor member).
+- **Privacy group**:   The actors that are involved in a set of transactions.
+
+
+### How Private Transaction is Processed
+
+1. DApp sends a private transaction through [eea_sendrawtransaction](../Reference/JSON-RPC-API-Methods.md#eea_sendRawTransaction),
+  uses the `privateFor` attribute to specify the list of actors,
+  `privateFrom` with the sender public key, and `restriction` to handle
+  with *restricted* and *unrestricted* transaction.
```

Comment:
It's stated below that `unrestricted` private transactions are not supported so why would it be specified?